pkg/controller/nodes/nodes_controller.go

```diff
@@ -14,7 +14,7 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
+	schema "k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
You can leave it as it was, it doesn't have any effect.

```suggestion
	"k8s.io/apimachinery/pkg/runtime/schema"
```